bedtools_merge_v2.28.0_cv2/bedtools_merge.cwl

```diff
@@ -0,0 +1,80 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+id: bedtools_merge
+baseCommand:
+  - bedtools
+  - merge
+inputs:
+  - id: input
+    type: File
+    inputBinding:
+      position: 5
+      prefix: '-i'
+      shellQuote: false
+    doc: BEDgraph format file generated from Bedtools Genomecov module
+  - id: output_file_name
+    type: string
+  - id: memory_overhead
+    type: int?
+  - id: memory_per_job
+    type: int?
+  - id: number_of_threads
+    type: int?
+  - default: 0
+    id: distance_between_features
+    type: int?
+    inputBinding:
+      position: 0
+      prefix: '-d'
+      shellQuote: false
+    doc: Maximum distance between features allowed for features to be merged.
+outputs:
+  - id: output_file
+    type: File?
+    outputBinding:
+      glob: |2-
+          ${
+            if (inputs.output_file_name.length>0)
```

Comment:
Why are we looking at the length of the file❓

Reply:
to check for an **empty** string in the input YAML